cinder/volume/drivers/rbd.py

```
@@ -562,8 +562,13 @@ def _delete_backup_snaps(self, rbd_image, volume_name, context):
         backup_snaps = self._get_backup_snaps(rbd_image)
         if backup_snaps:
             ctxt = context.elevated()
+            backup = None
             for snap in backup_snaps:
-                backup = self.db.backup_get(ctxt, snap['backup_id'])
+                try:
+                    backup = self.db.backup_get(ctxt, snap['backup_id'])
+                except exception.NotFound:
+                    backup = None
+                    pass
```

I am trying to suppress the exception, hence the "pass"